Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -9,8 +9,8 @@
 
 namespace OxyPlot.Series
 {
-    using System;
-    using System.Collections.Generic;
+	using System;
```

Comment:
SA1027 violation. Please follow the current coding style! :-) Otherwise I need to make a commit simply to fix the coding style...